sentry/src/main/java/io/sentry/HttpStatusCodeRange.java

```diff
@@ -0,0 +1,29 @@
+package io.sentry;
+
+/**
+ * The Http status code range. Example for a range: 400 to 499, 500 to 599, 400 to 599 The range is
+ * inclusive so the min and max is considered part of the range.
+ *
+ * <p>Example for a single status code 400, 500
+ */
+public final class HttpStatusCodeRange {
+  public static final int DEFAULT_MIN = 500;
+  public static final int DEFAULT_MAX = 599;
+
+  private final int min;
+  private final int max;
+
+  public HttpStatusCodeRange(final int min, final int max) {
+    this.min = min;
+    this.max = max;
+  }
+
+  public HttpStatusCodeRange(final int statusCode) {
```

Comment:
`m`: I think we don't need that. A range can also be from 1 to 1. This would simplify the class.

Reply:
Forcing someone to `HttpStatusCodeRange(1, 1)` is a bad UX, @romtsn what do you think?